server/src/test/java/org/elasticsearch/rest/action/cat/RestNodesActionTests.java

```diff
@@ -69,15 +70,14 @@ public void testBuildTableDoesNotThrowGivenNullNodeInfoAndStats() {
         action.buildTable(false, new FakeRestRequest(), clusterStateResponse, nodesInfoResponse, nodesStatsResponse);
     }
 
-    public void testCatNodesWithLocalDeprecationWarning() {
+    public void testCatNodesRejectsLocalParameter() {
+        assumeTrue("test is only needed in v8, can be removed in v9", Version.CURRENT.major == Version.V_7_0_0.major + 1);
```

Comment:
Wouldn't we want to `assertTrue` here so that this starts failing on the version bump and we can remove it instaed of having it live on as dead code in a skipped test?

Reply:
I figured we could defer this until we drop the v7 version constants, which normally happens some time after moving to v9. The compile error from `Version.V_7_0_0.major ` will limit how long this lives.